tools/monitoring/criersub/subscriber.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package criersub
```

Comment:
rename subscriber? This should not be tied to crier

Reply:
Done. The original idea of prefixing crier is because it will be sort of tied to crier after it uses the ReportMessage defined in the crier reporter once I figured out the dependency.